src/khoj/interface/web/chat.html

```diff
@@ -335,7 +348,14 @@
             box-shadow: 0 0 11px #aaa;
         }
         #chat-input {
-            font-size: medium;
+            font-family: roboto, karma, segoe ui, sans-serif;
```

Comment:
`font-family` seems to be already set for the chat.html body, so we may not need this here?

Reply:
The original input box can derive `font-family`, but if I replace it with `textarea`, it can't. Did I miss something ?